pkg/services/rediscache/provision.go

```diff
@@ -77,5 +73,58 @@ func (s *serviceManager) deployARMTemplate(
 	}
 	dt.PrimaryKey = service.SecureString(primaryKey)
 
+	if instance.ProvisioningParameters.GetString("enableNonSslPort") == enabled {
```

Comment:
This can be simplified to a single assignment

```go
dt.NonSSLEnabled = (instance.ProvisioningParameters.GetString("enableNonSslPort") == enabled)
```

Reply:
That's neat. I will change it later.